test/policies/rate-limiter/rate-limit.by.host.test.js

```diff
@@ -25,7 +25,7 @@ describe('rate-limit by host', () => {
                 action: {
                   max: 1,
                   // eslint-disable-next-line no-template-curly-in-string
-                  rateLimitBy: '${req.host}'
+                  rateLimitBy: '${req.hostname}'
```

Comment:
If you are into hostname, please make changes as fix for #432 then
technically, since we are listening on the single port 
hostname is equivalent to host

the only difference would be if you want different rules for https vs http

`host` should allow that separation. but that is highly unlikely scenario